datareservoirio/storage/storage.py

```diff
@@ -15,6 +15,14 @@
 
 log = logging.getLogger(__name__)
 
+_AZURE_SESSION = requests.Session()

```

Comment:
Is this really a blobstorage or blobstorageaccount session? All of our resources are in azure.

Reply:
Do you mean we should call it `BLOBSTORAGE_SESSION` to be more precise? :)

Reply:
Just keep the underscore prefix, and call it whatever you like :)

Reply:
renamed to BLOBSTORAGE_SESSION now